app/components/teachers/details/past_induction_periods_component.html.erb

```diff
@@ -4,7 +4,7 @@
   <% past_periods.each do |period| %>
     <li>
       <%=
-        govuk_summary_list(card: { title: period.appropriate_body.name, actions: can_edit?(period) ? [edit_link(period)] : [] }) do |sl|
+        govuk_summary_list(card: { title: period.appropriate_body.name, actions: enable_edit == true ? [edit_link(period)] : [] }) do |sl|
```

Comment:
Is this another one of those situations where we need to split out the admin version of a component? So the admin version would have editing enabled?